### ISSUE TITLE
Report when files are missing instead of reporting no issue

### ISSUE THREAD

**Issue by ptalbert** [ryran/xsos#184]
xsos --ethtool will report [None] for Interface Errors if the ethtool_-S_{$DEVICE} files are missing. This is seen on some older RHEL 5 series SOS Reports.

Ideally, xsos would report [No available stats files] or some such output instead of reporting there are no errors as this is misleading.

**Comment by ryran** [comment]
Ahh yes I forgot about this. Good call.

**Comment by ryran** [comment]
Fixed in forthcoming 0.7.7.

```
$ xsos -e .
ETHTOOL
  Interface Status:
    bond0                link=up 10000Mb/s full (autoneg=N)  rx ring UNKNOWN    drv bonding v3.7.1 / fw 2
    bond1                link=up 1000Mb/s full (autoneg=N)   rx ring UNKNOWN    drv bonding v3.7.1 / fw 2
    eth0   0000:03:00.0  link=DOWN                           rx ring 200/2047   drv tg3 v3.137 / fw 5719-v1.38 NCSI v1.2.37.0
    eth1   0000:03:00.1  link=DOWN                           rx ring 200/2047   drv tg3 v3.137 / fw 5719-v1.38 NCSI v1.2.37.0
    eth2   0000:03:00.2  link=up 1000Mb/s full (autoneg=Y)   rx ring 200/2047   drv tg3 v3.137 / fw 5719-v1.38 NCSI v1.2.37.0
    eth3   0000:03:00.3  link=up 1000Mb/s full (autoneg=Y)   rx ring 200/2047   drv tg3 v3.137 / fw 5719-v1.38 NCSI v1.2.37.0
    eth4   0000:04:00.0  link=up 10000Mb/s full (autoneg=N)  rx ring 1024/1024  drv be2net v10.4r / fw 10.2.340.19
    eth5   0000:04:00.1  link=up 10000Mb/s full (autoneg=N)  rx ring 1024/1024  drv be2net v10.4r / fw 10.2.340.19
  Interface Errors:
    eth4  Missing ethtool_-S file
    - - - - - - - - - - - - - - - - - - -
    eth5  rx_udp_checksum_errs: 1
```
